spacemesh/v2alpha1/activation.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package spacemesh.v2alpha1;
+
+message Activation {
+    oneof versioned {
+        ActivationV1 v1 = 1;
+    }
+}
+
+message ActivationV1 {
+    bytes id = 1;
+    bytes node_id = 2;
+    bytes signature = 3;
+    uint32 publish_epoch = 4;
+    uint64 sequence = 5;
+    bytes prev_atx = 6;
+    bytes positioning_atx = 7;
+    bytes committment_atx = 8;
+    Post initial_post = 9;
+    string coinbase = 10;
+    uint32 units = 11;
+    uint32 base_height = 12;
+    uint32 ticks = 13;
+    PoetProof poet_proof = 14;
+    Post post = 15;
+    PostMeta post_meta = 16;
+    VRFPostIndex vrf_post_index = 17;
+}
+
+message VRFPostIndex {
+    uint64 nonce = 1;
+}
+
+message PoetProof {
+    repeated bytes proof_nodes = 1;
+    uint64 leaf = 2;
+}
+
+message PostMeta {
+    bytes challenge = 1;
+    uint64 labels = 2;
```

Comment:
Is it the `"labelsPerUnit"`? If so, I think it's not required here as this is a network parameter (all ATXs on the network have the same value). I know it is part of the Activation structure (it probably shouldn't be) so I guess its fine to keep it here for symmetry :shrug: .